src/api.ts

```diff
@@ -92,3 +99,33 @@ function mapDepartureQueryKeys(
     favorites: quaysWithDepartures,
   };
 }
+
+type FetchTripArgs = {
+  originId: string;
+  destinationId: string;
+  arriveBy: boolean;
+  dateTime: Date;
+  pageCursor: string;
+};
+
+export async function fetchTrip({
+  originId,
+  destinationId,
+  arriveBy,
+  dateTime,
+  pageCursor,
+  signal,
+}: FetchTripArgs & { signal?: AbortSignal }) {
+  return await fetchJourneyPlannerData<TripQuery>(
+    getJourneyQuery(),
+    {
+      fromPlace: originId,
+      toPlace: destinationId,
+      numTripPatterns: 10,
```

Comment:
I don't think we need to set `numTripPatterns` for our use case. Showing more than 10 suggestions shouldn't be a problem, and I'm more in favor of relying on defaults. 

From Enturs docs:

> The maximum number of trip patterns to return. Note! This reduce the number of trip patterns AFTER the OTP travel search is done in a post-filtering process. There is little/no performance gain in reducing the number of trip patterns returned.

```suggestion
```